### PR TITLE
WCAG upgrade - Create test page for: Color (foreground/background) (Revised)

### PR DESCRIPTION
**Pull request by BrahimMahadi**
This pull request includes a test page with working examples filled with lorem ipsum text for Utilities/helpers. More info can be found by referencing WET-319.

### CODE REVIEW

File: common/colour/colour-fr.html [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+	<div class="well mrgn-tp-md mrgn-bttm-md text-center">
+		<h5 class="mrgn-tp-md">Aligné au centre</h5>
+		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+	</div>
+	<div class="well mrgn-tp-md mrgn-bttm-md text-justify">
+		<h5 class="mrgn-tp-md">Aligné-justifier</h5>
+		<p>Ambitioni dedisse scripsisse iudicaretur. Cras mattis iudicium purus sit amet fermentum. Donec sed odio operae, eu vulputate felis rhoncus. Praeterea iter est quasdam res quas ex communi. At nos hinc posthac, sitientis piros Afros. Petierunt uti sibi concilium totius Galliae in diem certam indicere. Cras mattis iudicium purus sit amet fermentum.</p>
+	</div>
+	<div class="well mrgn-tp-md mrgn-bttm-md text-nowrap" style="width: 8rem; border: 2px solid black">
+		<h5 class="mrgn-tp-md">Pas d'emballage</h5>
```
Comment (duboisp) [inline common/colour/colour-fr.html:184]: "Emballage" n'est pas le terme approprié ici. Voir les mots alternatifs sur Termium Plus.

**Review by duboisp — CHANGES_REQUESTED** [review]
I completed a review.

* [ ] Please apply the suggested change
* [ ] Apply the requested change for English also on the French equivalent page
* [ ] Update each appropriate source code sample
* [ ] Ajuster la traduction du mot "wrap" par rapport à son contexte d'utilisation

Once the above is fixed, I will do another review.

Think that is not required for this PR, but for future consideration:
* Re-think the mise en évidence of some section. Is using the well is accurate or should we create a component that do some content indentation with a white background to show it is apart of the main content of the page?
* The use of well may not optimal to show the user of text color, this presentation could benefit of a review/re-think.
* (Just an idea) May be use an explicit section element for the "example" h4 implicit section and use expand/collapse for source code.

File: common/colour/colour-en.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+	&lt;div class="<strong>bg-gctheme</strong> text-white well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Heading&lt;/h5&gt;
+		&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ante nulla, condimentum at mollis et, scelerisque quis dolor. Suspendisse viverra in ante id consequat. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce sodales ipsum vitae faucibus iaculis. Nulla in blandit dolor. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aenean non interdum sapien. Nunc non volutpat dui, in malesuada justo. Etiam feugiat accumsan pulvinar. Quisque blandit est in placerat finibus. Sed et ullamcorper velit. Nullam euismod iaculis ligula sit amet consequat. Aenean luctus quis nisi nec feugiat.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3><code>bg-cover</code></h3>
+<p>Set background image to fully cover the box size.</p>
+<h4>Working example</h4>
+<div class="bg-cover well" data-bgimg="../../demos/tabs/img/investinourfuture.jpg">
```
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply common/colour/colour-en.html]: Addressed in commit "Addressed Review Comments"

File: common/colour/colour-en.html [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+		&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h2 id="gcweb">GCWeb</h2>
+
+<h3><code>bg-gctheme</code></h3>
+<p>Set background to the gctheme.</p>
+<h4>Working example</h4>
+<div class="bg-cover well" data-bgimg="../../demos/tabs/img/investinourfuture.jpg">
```
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply common/colour/colour-en.html]: Addressed in commit "Addressed Review Comments"

File: common/colour/colour-en.html [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Heading&lt;/h5&gt;
+		&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ante nulla, condimentum at mollis et, scelerisque quis dolor. Suspendisse viverra in ante id consequat. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce sodales ipsum vitae faucibus iaculis. Nulla in blandit dolor. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aenean non interdum sapien. Nunc non volutpat dui, in malesuada justo. Etiam feugiat accumsan pulvinar. Quisque blandit est in placerat finibus. Sed et ullamcorper velit. Nullam euismod iaculis ligula sit amet consequat. Aenean luctus quis nisi nec feugiat.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3><code>bg-center</code></h3>
+<p>Set background image in the center of the container.</p>
+<h4>Working example</h4>
+<div class="bg-center well" data-bgimg="../../demos/tabs/img/investinourfuture.jpg">
```
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply common/colour/colour-en.html]: Addressed in commit "Addressed Review Comments"

File: common/colour/colour-en.html [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Heading&lt;/h5&gt;
+		&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ante nulla, condimentum at mollis et, scelerisque quis dolor. Suspendisse viverra in ante id consequat. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce sodales ipsum vitae faucibus iaculis. Nulla in blandit dolor. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aenean non interdum sapien. Nunc non volutpat dui, in malesuada justo. Etiam feugiat accumsan pulvinar. Quisque blandit est in placerat finibus. Sed et ullamcorper velit. Nullam euismod iaculis ligula sit amet consequat. Aenean luctus quis nisi nec feugiat.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3><code>bg-darker</code></h3>
+<p>Set a black background to an element.</p>
+<h4>Working example</h4>
+<div class="bg-cover well" data-bgimg="../../demos/tabs/img/investinourfuture.jpg">
```
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply common/colour/colour-en.html]: Addressed in commit "Addressed Review Comments"

File: common/colour/colour-en.html [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+		&lt;h5 class="mrgn-tp-md"&gt;Heading&lt;/h5&gt;
+		&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ante nulla, condimentum at mollis et, scelerisque quis dolor. Suspendisse viverra in ante id consequat. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce sodales ipsum vitae faucibus iaculis. Nulla in blandit dolor. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aenean non interdum sapien. Nunc non volutpat dui, in malesuada justo. Etiam feugiat accumsan pulvinar. Quisque blandit est in placerat finibus. Sed et ullamcorper velit. Nullam euismod iaculis ligula sit amet consequat. Aenean luctus quis nisi nec feugiat.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3><code>bg-center</code></h3>
+<p>Set background image in the center of the container.</p>
+<h4>Working example</h4>
+<div class="bg-center well" data-bgimg="../../demos/tabs/img/investinourfuture.jpg">
+	<div class="well mrgn-tp-md mrgn-bttm-md">
```
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply common/colour/colour-en.html]: Addressed in commit "Addressed Review Comments"

File: common/colour/colour-en.html [hunk 7]
```diff
@@ -0,0 +1,314 @@
+---
+{
+	"title": "Color (Foreground/Background)",
+	"language": "en",
+	"description": "Colors page including working examples to test how various text appears on different backgrounds.",
+	"dateModified": "2023-04-18"
```
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply common/colour/colour-en.html]: Addressed in commit "Addressed Review Comments"

File: common/colour/colour-fr.html (hunk 1 above)
Comment (BrahimMahadi) [reply common/colour/colour-fr.html]: Addressed in commit "Addressed Review Comments"

**Review by duboisp — CHANGES_REQUESTED** [review]
2 little change and we should be good to merge once fixed.

Optionally you can remove this page as it's going to be duplicated by this one: https://wet-boew.github.io/GCWeb/sites/helpers/colour-en.html

File: common/colour/colour-en.html [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+<h4>Code sample</h4>
+<pre><code>&lt;div class="<strong>bg-norepeat</strong> well" data-bgimg="https://wet-boew.github.io/wet-boew/demos/tabs/img/investinourfuture.jpg"&gt;
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Heading&lt;/h5&gt;
+		&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ante nulla, condimentum at mollis et, scelerisque quis dolor. Suspendisse viverra in ante id consequat. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce sodales ipsum vitae faucibus iaculis. Nulla in blandit dolor. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aenean non interdum sapien. Nunc non volutpat dui, in malesuada justo. Etiam feugiat accumsan pulvinar. Quisque blandit est in placerat finibus. Sed et ullamcorper velit. Nullam euismod iaculis ligula sit amet consequat. Aenean luctus quis nisi nec feugiat.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3><code>bg-darker</code></h3>
+<p>Set a black background to an element.</p>
```
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply common/colour/colour-en.html:280]: Changes made in the commit "Added bg-gctheme".

File: common/colour/colour-fr.html [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Titre&lt;/h5&gt;
+		&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ante nulla, condimentum at mollis et, scelerisque quis dolor. Suspendisse viverra in ante id consequat. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce sodales ipsum vitae faucibus iaculis. Nulla in blandit dolor. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aenean non interdum sapien. Nunc non volutpat dui, in malesuada justo. Etiam feugiat accumsan pulvinar. Quisque blandit est in placerat finibus. Sed et ullamcorper velit. Nullam euismod iaculis ligula sit amet consequat. Aenean luctus quis nisi nec feugiat.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3><code>bg-darker</code></h3>
+<p>Définissez un arrière-plan noir sur un élément.</p>
+<h4>Exemple</h4>
+<div class="well mrgn-tp-md mrgn-bttm-md bg-darker text-white">
```
[the remark this answers is not in the document]
Comment (BrahimMahadi) [reply common/colour/colour-fr.html]: Changes made in the commit "Added bg-gctheme".

**Review by duboisp — APPROVED** [review]
Great work,

This is only a content change.

**Pull request merged**